inference-engine/src/vpu/graph_transformer/src/stages/gather_elements.cpp

```diff
@@ -47,7 +47,8 @@ class GatherElementsStage final : public StageNode {
     getBatchSupportInfoImpl(StageDataInfo<BatchSupport> &batchInfo) override {}
 
     StageSHAVEsRequirements getSHAVEsRequirementsImpl() const override {
-        return StageSHAVEsRequirements::NotNeeded;
+        const auto inner_dim_axis = attrs().get<bool>("inner_dim_axis");
+        return (inner_dim_axis ? StageSHAVEsRequirements::NeedMax : StageSHAVEsRequirements::NotNeeded);
```

Comment:
Such a change does not reflect PR name `add functional tests`. It changes behavior as well..